src/mp_api/client.py

```diff
@@ -114,6 +122,7 @@ def __init__(
         self.session = BaseRester._create_session(
             api_key=api_key, include_user_agent=include_user_agent
         )
+        self.contribs = Client()
```

Comment:
Should pass the API key along to `Client()` here

Reply:
Done. Can you make sure I did it correctly?